minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -333,17 +279,16 @@ private Process _exec(Class<?> clazz, List<String> extraJvmOpts, String... args)
     log.debug("Starting MiniAccumuloCluster process with class: " + clazz.getSimpleName()
         + "\n, jvmOpts: " + extraJvmOpts + "\n, classpath: " + classpath + "\n, args: " + argList
         + "\n, environment: " + builder.environment());
-    Process process = builder.start();
 
-    LogWriter lw;
-    lw = new LogWriter(process.getErrorStream(),
-        new File(config.getLogDir(), clazz.getSimpleName() + "_" + process.hashCode() + ".err"));
-    logWriters.add(lw);
-    lw.start();
-    lw = new LogWriter(process.getInputStream(),
-        new File(config.getLogDir(), clazz.getSimpleName() + "_" + process.hashCode() + ".out"));
-    logWriters.add(lw);
-    lw.start();
+    Integer hashcode = builder.hashCode();
+
+    builder = builder
+        .redirectError(new File(config.getLogDir(),
+            clazz.getSimpleName() + "_" + hashcode.toString() + ".err"))
```

Comment:
I just noticed the call to `toString()` on `hashcode` is unnecessary.  I can fix this when I merge.